text/rustconf-steering-committee.md

```diff
@@ -5,46 +5,49 @@
+The Rust Foundation staff team is proposing the formation of a RustConf Steering Committee with the following parameters:
+- Members: 4 total members from the Rust Project -- one leadership council member, one Project Director member, and two members from the wider Rust community. We feel that an even split between leadership members and community members makes sense as the former positions can provide context and Project authority while the latter can advocate for those who are not necessarily in a position of power within the Project. We will lean on the Leadership Council as a whole to select this group based on a process of self-nomination initiated by the Foundation and promoted by the Leadership Council.  
+- Scope: The Foundation would like the Steering Committee's involvement in the following areas:
+ > - **Program Committee:** The Steering Committee should establish the structure, personnel, and processes of the Program Committee with oversight and collaboration from the Rust Foundation. The Foundation will take this plan from the Steering Committee to formally set up the Program Committee which will select talks after the closure of the call for talk proposals. Steering Committee members will not be on the Program Committee or be involved in selecting talks, although they can advise the Program Committee on key priorities as talks are being selected. To stay on schedule, the Foundation will likely need to open the CFP earlier than these two committees are in place, however, we will share our intentions for talk tracks/suggested topics with the Leadership Council in advance for feedback. 
+>  - **UnConf Day:** The Foundation would be pleased to offer an UnConf Day again in 2024, but would lean on the Steering Committee to decide this. If they decide there should be an UnConf Day after RustConf, the Foundation would like the Steering Committee to provide us with a basic plan including content and identification of people from the Project we should be logistically coordinating with ("UnConf Day Organizers") to ensure proper supplies and resources are available to UnConf organizers and attendees. The Foundation would welcome the Steering Committee's participation in planning and running the UnConf alongside the Organizers they identify.
+> - **Other considerations:** The Foundation would also like to lean on the Steering Committee to understand if there are any other areas of the event they would like to be involved in on behalf of the Rust Project. The Rust Foundation needs to operate independently on major event decisions (like co-located events and conference activities, tickets/access, general event experience, and sponsorship experience) but will welcome feedback from the Steering Committee along the way.
```

Comment:
> The Foundation would also like to lean on the Steering Committee to understand if there are any other areas of the event they would like to be involved in on behalf of the Rust Project.

This seems wishy-washy in a way that is unhelpful. I think what you're trying to say is that the various groups should make a best effort to involve input from other groups in their areas of responsibility. I think we should state that.

> All responsible groups should make an active effort to involve the input of other groups in their areas of responsibility. For example, the Foundation Staff should coordinate with the steering committee on event location before making their ultimate decision. Likewise, the Steering Committee should involve Foundation Staff in discussions on supplementary events even before they make a final decisions to minimize difficulties with logistics planning. 

> he Rust Foundation needs to operate independently on major event decisions (like co-located events and conference activities, tickets/access, general event experience, and sponsorship experience)

You've stated this mater of factly, but I'm not sure I agree that the Staff *must* be able to act independently in all of these areas in order to effectively organize RustConf. While logisical concerns like Sponsorship experience and tickets/access seem to fall squarely in the "operations" bucket (and thus appropriate for the Staff to be charged with executing on them more or less independently), "co-located events", "conference activities", and "general event experience" seem to be much more of a programming concern, and thus I would have imagined they would have falled squarely in the responsibility of the Steering Committee.